### PR TITLE
verbose should register a api_log_handler

### PR DESCRIPTION
**Pull request by asmacdo**
https://pulp.plan.io/issues/145

### CODE REVIEW

**Comment by dkliban** [comment]
ok test

File: client_lib/pulp/client/launcher.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def _create_bindings(config, cli_logger, username, password, verbose=None):
         cert_filename = None
 
     api_logger = None
-    if verbose and verbose > 1:
+    if verbose and verbose >= 1:
```
Comment (rbarlow) [inline client_lib/pulp/client/launcher.py]: It was my understanding that we only wanted to log API output with -vv and not with just -v.

**Comment by rbarlow** [comment]
I don't think we want to log the API calls with -v, and the Redmine issue doesn't seem to indicate this either. What non-API info() calls are there during a sync?

**Comment by ipanova** [comment]
@rbarlow when i've been reviewing the PR that introduced this functionality a had same question
https://github.com/pulp/pulp/pull/1698#issuecomment-77535323
@asmacdo maybe this will help to clear the situation.

**Comment by asmacdo** [comment]
This [failed QA](https://pulp.plan.io/issues/145#note-27), but based on your comments, this looks like the correct behavior. Do you agree then that the issue should go back to QA and that this PR should be closed? @ipanova @rbarlow

**Comment by ipanova** [comment]
@asmacdo i am ok to close this pr, if we add to bug description the info mentioned above.
let's see what @rbarlow  will say.

**Comment by rbarlow** [comment]
@asmacdo @ipanova I think we should put it back to ON_QA and close this.

**Pull request closed without merging**